content/en/docs/instrumentation/cpp/exporters.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Exporters
+weight: 4
+---
+
+In order to visualize and analyze your
+[traces](/docs/concepts/signals/traces/#tracing-in-opentelemetry) and metrics,
+you will need to export them to a backend.
+
+## Logging exporter
```

Comment:
Is it called the logging exporter for C++ devs? Curious since the class name isn't that.

Reply:
It's called OStream Exporter, with default output to stdout. I guess something like Console / OStream exporter would sound better.